esp/esp/program/tests.py

```diff
@@ -341,8 +342,6 @@ def makeprogram(self):
                 'teacher_reg_end':   '3001-01-01 00:00:00',
                 'student_reg_start': '2000-01-01 00:00:00',
                 'student_reg_end':   '3001-01-01 00:00:00',
-                'publish_start':     '2000-01-01 00:00:00',
-                'publish_end':       '3001-01-01 00:00:00',
```

Comment:
Why remove these fields?

Reply:
They don't exist anymore in the form.

Reply:
Ok. Can we get rid of the remaining references in `esp/esp/program/tests.py` and also all the commented out reference in `esp/esp/program/forms.py` lines 155-174?